packages/account/src/Components/self-exclusion/__tests__/self-exclusion-confirm-page.spec.tsx

```diff
@@ -28,7 +28,6 @@ describe('<SelfExclusionConfirmPage />', () => {
         is_app_settings: false,
         is_wrapper_bypassed: false,
         toggleArticle: jest.fn(),
-        is_uk: false,
```

Comment:
@matin-deriv Are we removing these countries because we no longer have clients there?

Reply:
UK, Belgium and Isle of Man - yes.
Any condition for other countries need to be handled by the API.

Reply:
Okay 🙏